### PR TITLE
auth: Convert logrus to slog

### PR DESCRIPTION
**Pull request by sayboras**

### CODE REVIEW

File: operator/auth/spire/log_wrapper.go [hunk 1]
```diff
@@ -3,35 +3,37 @@
...
 func (l *spiffeLogWrapper) Infof(format string, args ...interface{}) {
-	l.log.Infof(format, args...)
+	l.log.Info(format, args...)
 }
 
 // Warnf logs a warning message
 func (l *spiffeLogWrapper) Warnf(format string, args ...interface{}) {
-	l.log.Warnf(format, args...)
+	l.log.Warn(format, args...)
 }
```
Comment (tklauser) [inline operator/auth/spire/log_wrapper.go]: These will now panic if `len(args) % 2 != 0` as they expect `args` to be a set of key-value pairs. Should we instead convert these to use `fmt.Sprintf` to maintain the current behavior?

File: operator/auth/spire/log_wrapper.go [hunk 2]
```diff
@@ -40,5 +42,5 @@ func (l *spiffeLogWrapper) Warnf(format string, args ...interface{}) {
 // result in an error passed back to the function caller which then is logged
 // as an error.
 func (l *spiffeLogWrapper) Errorf(format string, args ...interface{}) {
-	l.log.Warnf(format, args...)
+	l.log.Warn(format, args...)
```
Comment (tklauser) [inline operator/auth/spire/log_wrapper.go]: Same here, should we use `fmt.Sprintf` to format the log string?

File: operator/auth/spire/log_wrapper.go (hunk 1 above)
Comment (sayboras) [reply operator/auth/spire/log_wrapper.go]: I don't think panic will happen (i.e. some predefined string will be used), but it's good point to change to fmt.Sprintf :+1:

**Review by tklauser — APPROVED** [review]
Thanks

**Review by youngnick — APPROVED** [review]

**Pull request merged**